### PR TITLE
0.11.0 Changelog and Pins

### PR DESCRIPTION
**Pull request by Lnaden**
## Description

Client limits are now 0.11.0 <= Client <= 0.11.99 (only checks the X.Y)
Pins for Elemental: 0.9 (not released)
Pins for Engine: 0.11 (not released)

Missing placeholder issue from @dgasmith 

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [ ] Replace Placeholder issue in the changelog
  - [ ] Test this PR once Engine 0.11 and Elemental 0.9 is released

## Status
- [x] Changelog updated
- [ ] Ready to go

### CODE REVIEW

**Comment by dgasmith** [comment]
Placeholder issue is up.

File: docs/qcfractal/source/changelog.rst [hunk 1]
```diff
@@ -15,36 +15,47 @@ Changelog
...
+  method.
+  Those fast queries are now used in ``torsiondrive.get_final_molecules`` and ``torsiondrive.get_final_results``. More
+  Advanced queries will be added.
 - (:pr:`394`) Adds ``tag`` and ``manager`` selector fields to ``client.query_tasks``.
   This is helpful for managing jobs in the queue and detecting failures.
-
-- (:pr:`400`) Adds Dockerfiles corresponding to builds on `Docker Hub <https://cloud.docker.com/u/molssi/repository/list>`_.
+- (:pr:`400`, :pr:`401`, :pr:`410`) Adds Dockerfiles corresponding to builds on
+  `Docker Hub <https://cloud.docker.com/u/molssi/repository/list>`_.
+- (:pr:`406`) The ``Datset`` collection's primary indices (database level) have been updated to reflect its new
```
Comment (mattwelborn) [inline docs/qcfractal/source/changelog.rst]: Datset -> Dataset

File: docs/qcportal/source/changelog.rst [hunk 2]
```diff
@@ -15,6 +15,52 @@ Changelog
...
+  ``get_molecules`` returns the ``Molecule`` associated with a dataset.
+- (:pr:`393`) A new feature added to ``Client`` to be able to have more custom and fast queries, the ``custom_query``
+  method.
+  Those fast queries are now used in ``torsiondrive.get_final_molecules`` and ``torsiondrive.get_final_results``. More
+  Advanced queries will be added.
+- (:pr:`394`) Adds ``tag`` and ``manager`` selector fields to ``client.query_tasks``.
+  This is helpful for managing jobs in the queue and detecting failures.
+- (:pr:`400`, :pr:`401`, :pr:`410`) Adds Dockerfiles corresponding to builds on
+  `Docker Hub <https://cloud.docker.com/u/molssi/repository/list>`_.
+- (:pr:`406`) The ``Datset`` collection's primary indices (database level) have been updated to reflect its new
```
Comment (mattwelborn) [inline docs/qcportal/source/changelog.rst]: Ditto Datset

**Comment by dgasmith** [comment]
@Lnaden Please rebase when you can! (no rush)

**Comment by Lnaden** [comment]
okay, that should be the rebase

**Pull request merged**